stories/topics/proc-aws-generate-upgrade-data-file.adoc

```diff
@@ -2,27 +2,35 @@
 
 = Generating the data file
 
-The commands in this section create a directory and populate it with an empty data template that, when populated, is used during the upgrade. 
+The commands in this section create a directory and populate it with an empty data template that, when populated, is used during the upgrade.
 
 .Procedure
 
-. Run the following commands to generate the required data file. 
+. Run the following commands to generate the required data file.
 +
 [literal, options="nowrap" subs="+attributes"]
 ----
 # Create a folder to hold the configuration files
 $ mkdir command_generator_data
 ----
 . Populate the `command_generator_data` folder with the configuration file template.
++
+
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:tech-note-linux-files-owned-by-root[tech notes] for more information.
```

Comment:
generator is owned by....

For more information, read the xref:tech-note-linux-files-owned-by-root[tech notes].